components/expenses/ExpenseForm.js

```diff
@@ -570,6 +576,30 @@ const ExpenseFormBody = ({
                     {errors.payoutMethod.data.currency.toString()}
                   </Box>
                 )}
+                <StyledHr flex="1" borderColor="#ffffff" mx={2} />
+                <StyledButton
+                  type="button"
+                  buttonStyle="borderless"
+                  width={['100%', 'auto']}
+                  color="red.500"
+                  mt={1}
+                  mx={[2, 0]}
+                  mr={[null, 3]}
+                  whiteSpace="nowrap"
+                  onClick={() => {
+                    if (window.confirm(formatMessage(msg.confirmResetExpense))) {
+                      handleReset();
+                      setStep(STEPS.PAYEE);
+                      if (formPersister) {
+                        formPersister.clearValues();
+                        window.scrollTo(0, 0);
+                      }
```

Comment:
I think this has some problems. My testing suggest this doesn't seem to clear the form; I haven't investigated into the root cause yet though. 🤔 

![expense-reset](https://user-images.githubusercontent.com/12435965/135703700-acec17e7-8f5b-4f02-8e80-3dce75d8df60.gif)

Reply:
Could be browser specific? Is this firefox?

Reply:
Yes it is Firefox. And yes it might be Firefox specific; one of the reasons I only use Firefox; I find it's nifty at finding more bugs. 😉  😉 The specific version is, 92.0 (64-bit).

Reply:
Just tested locally on `firefox 92.0 (64-bit)`, working fine for me. I am using `macOS`.

<details>

<summary>   Editing an expense  </summary>

https://user-images.githubusercontent.com/46647141/135737096-5a06c6ff-26ed-413f-98e4-444bfc4e8552.mp4
</details>

<details>

<summary>   Creating an expense  </summary>

https://user-images.githubusercontent.com/46647141/135737146-f6642f08-1e59-4ad9-aa66-569891a8e395.mp4
</details>





Did you go ahead to the third step while creating a new expense, i.e, review the expense and clicked edit expense there instead?

<img width="797" alt="Screenshot 2021-10-03 at 7 49 40 AM" src="https://user-images.githubusercontent.com/46647141/135737183-7fc744fe-8aee-44e5-bf29-41f3a80408c8.png">

If that's the case then it will reset it to the previous state before editing just like the first case above which is while editing an expense.
In your shared GIF above if you update the amount from `$12` to `$200` and then hit the `reset form` button it will reset it to `$12` again.

<details>

<summary>   Demo  </summary>

https://user-images.githubusercontent.com/46647141/135737286-91bb7c48-60d5-488f-b56e-c5edf345acc3.mp4

</details>

Reply:
Yes sorry, I should have been more specific; the flow I carried out was just do an expense submission and go all the way to the final step, but don't submit the expense. Then click on the `Edit Expense` and then come back to the section where the reset button is. Let me show this with a screen-cast. 😄 

![reset-button-but](https://user-images.githubusercontent.com/12435965/135737670-bf91e2af-ee4d-4dc9-9adf-8e2d7c3a7f01.gif)

My thinking it it still needs to reset the form as the expense is not submitted and if we are displaying the reset button somewhere in the flow and the user clicks on it, it should reset the form. 😄

Reply:
@SudharakaP I have updated the PR as per your suggestion.

Here is the latest demo.

<details>
<summary>    Demo </summary>

https://user-images.githubusercontent.com/46647141/135781591-885fe35a-fda7-4091-bb96-b03f81301c0f.mp4
</details>

Reply:
Thanks. 👍🏽 Looks nice now. 😸